extension-compose/build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+  id("com.android.library")
+  kotlin("android")
+  id("org.jetbrains.dokka")
+  id("io.gitlab.arturbosch.detekt").version(Versions.detekt)
+}
+
+android {
+  compileSdk = AndroidVersions.Compose.compileSdkVersion
+  defaultConfig {
+    minSdk = AndroidVersions.Compose.minSdkVersion
+    targetSdk = AndroidVersions.Compose.targetSdkVersion
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  testOptions {
+    unitTests.apply {
+      isIncludeAndroidResources = true
+    }
+  }
+
+  composeOptions {
+    kotlinCompilerExtensionVersion = Versions.compose
+  }
+
+  buildFeatures {
+    compose = true
+  }
+
+  kotlinOptions {
+    freeCompilerArgs += "-Xexplicit-api=strict"
+  }
+}
+
+dependencies {
+  dependencies {
+    compileOnly(project(":sdk"))
+    api(Dependencies.googleMaterialDesign)
+    api(Dependencies.composeUi)
+    api(Dependencies.composeMaterial)
```

Comment:
Do we really need in `extension-compose` the material deps (`composeMaterial` and `googleMaterialDesign`).

Reply:
just did a clean up of dependencies, we for now only implement `composeUi` and `composeMaterial`, these are required to build the extension.